scenarioo-server/src/main/resources/config-for-demo/config.xml

```diff
@@ -28,6 +28,7 @@
         In case you want to contact the Scenarioo Development Team, please refer to the info popup tab 'Contacts'. We are looking forward to get your feedback. Thank you!
         &lt;/p&gt;
         &lt;p&gt;&lt;b&gt;Have fun!&lt;/b&gt;&lt;/p&gt;</applicationInformation>
+    <branchSelectionListOrder>name-ascending</branchSelectionListOrder>
```

Comment:
please remove and use the default for demo (to test that no one has to explicitly set it)